### PR TITLE
IotexShiba Metadata Request

### PR DESCRIPTION
**Pull request by IoTexShiba**
- **Project official website**: https://www.iotexshiba.io/

- **Project official social**: https://https://t.me/iotexshiba | https://twitter.com/IoTexShiba

### CODE REVIEW

**Review by simonerom — DISMISSED** [review]

**Comment by IoTexShiba** [comment]
@zimne1 thanks for approval. This account don't have write access. Could someone from dev team complete the merge? Thanks in regards

File: token-metadata.json [hunk 1]
```diff
@@ -7,6 +7,14 @@
     "symbol": "VITA",
     "decimals": 18
   },
+  "io186ngxd2tlrf4nnv7cmsgkkhv9ywhrkyq8rejue": {
+    "name": "IoTexShiba",
+    "description": "The first and fairest meme token on IoTex with great passive income rewards for holders.",
+    "logo": "io186ngxd2tlrf4nnv7cmsgkkhv9ywhrkyq8rejue.png",
+    "type": "xrc20",
+    "symbol": "IoShib",
+    "decimals": 18
```
Comment (ququzone) [inline token-metadata.json:16]: According contract's code  decimals is 9

Comment (IoTexShiba) [reply token-metadata.json]: sorry, my bad, thanks for pointing this out. I've updated now.

**Comment by IoTexShiba** [comment]
change with 9 decimals

**Review by ququzone — APPROVED** [review]

**Comment by IoTexShiba** [comment]
@zimne1 please look into this, approve and complete the merge as we don't have write access. thanks in advance.

**Pull request merged**